src/okapiActions.js

```diff
@@ -97,6 +104,7 @@ export { setCurrentUser,
   clearCurrentUser,
   setCurrentPerms,
   setLocale,
+  setTimezone,
```

Comment:
Don't you just hate that you have to put single lines of code in about eight different places when you add a new datum like this? I don't know what we could do differently to avoid it, but it does sometimes feel like Redux wants you to fill in forms in triplicate.